### ISSUE TITLE
Some drivers ignore lane arrows 

### ISSUE THREAD

**Issue by slyh** [slyh/Cities2-Various-Mods#11]
Work needs to be done to stop drivers from making “unsafe” manoeuvres.

**Comment by HarryBotter2333** [comment]
I agree, to be precise, there must be many people who want to 100% ban illegal U-turns on vehicles and the like. Of course, it should be adjustable from 0-100% at the back ?

Things like making a U-turn across a double solid line or making a U-turn at a narrow intersection are very troublesome :(

And I found that even if there is a "roundabout", people are too lazy to run to the roundabout and turn around, they will turn around as soon as they get the chance.

**Comment by slyh** [comment]
I spent some time looking into the code and was not able to find a way to ban those behaviours. We will have to live with that for now.

**Comment by timfrombriz** [comment]
Hey slyh,

In the release notes for 1.0.13f1 one of the changes was;
- Increased pathfind penalty for unsafe u-turns on highway

Im not sure if someone has the binaries for 1.0.13f1 and 1.0.12f1 releases (steam doesnt let you revert to old versions I dont think), if they were accessible, one could code diff the builds and see what was changed as a way to narrow down what were looking for given this was a change made between builds.

Also, Im a Unity C# developer, but never reversed compiled projects before.  Id love to get my fingers dirty on this project, but I've not used GitHub or done anything but solo work, and not sure how to get started unpacking the build and packing it pack up for testing let alone uploading confirmed changes.  Could I get a lending hand from you to get started on this, and if so, whats the best way to find you?  Send me a PM on here if you can help. Cheers

**Comment by slyh** [comment]
@timfrombriz That's an interesting finding. If you use Discord, you could join this [server](https://discord.gg/KGRNBbm5Fh) where some other people are also trying to work out the road network system. Head to the #reverse-engineering channel and look for the 'Networks' thread for the discussion.

If you don't use Discord, just let me know here, and I will reach out to you via email.
